docs/running-on-kubernetes.md

```diff
@@ -1356,6 +1356,26 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
   <td>3.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.job.minCPU</code></td>
```

Comment:
I'm just wondering if we can reuse some of the Spark configuration.

Reply:
Yep, this's a good question, there were some related use case of Spark with volcano in production can be shared in here:
- Case1 (this PR): `minCPU` = specified `minCPU`: for users who know their own cluster resources well, this is the basic user case.
- Case2: `minCPU` = driver.request + `executor.number` * `executor.request`: for users who don't care much about job resource usage.
- Case3: `minCPU` = (driver.request + `executor.number` * `executor.request`) * `factor`, for users want to guarantee the resources of the job in some level, but also want to improve the utilization of the cluster.

Reply:
also cc @holdenk

Reply:
We need to document what is used in case of `(none)`, @Yikun .

Reply:
```
If it is `(none)`,  indicates there are no limits on the minimum resources.
```

will add it